include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
> It is definitively a packaging issue, but it isn't a trivial issue. Autotools actually doesn't recommend installing the generated config.h in the /usr/local/include (which is where the library is installed).

True.  But I assumed that it should end up as `/usr/local/include/CppUTest/config.h` when installed, along with most of the other headers.  Then there wouldn't be any problem.